docs/tutorials/notebooks/tracking_progress.ipynb

```diff
@@ -7,21 +7,15 @@
     "tags": []
```

Comment:
<p>We're not passing it to its initializer (the word is overloaded and can also mean something else); would remove <code>to its initializer</code>  mention.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/351/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>